airbyte-integrations/connectors/source-clockify/metadata.yaml

```diff
@@ -1,47 +1,35 @@
+metadataSpecVersion: "1.0"
 data:
-  ab_internal:
-    ql: 100
-    sl: 100
   allowedHosts:
     hosts:
-      - api.clockify.me
+      - "api.clockify.me"
+  registryOverrides:
+    oss:
+      enabled: true
+    cloud:
+      enabled: true
+  remoteRegistries:
+    pypi:
+      enabled: false
+      packageName: airbyte-source-clockify
   connectorBuildOptions:
     baseImage: docker.io/airbyte/source-declarative-manifest:6.33.1@sha256:06468f2b0acdb0126a29757f67025f8f837014f70e3f079e10e304b0e1a6be4b
   connectorSubtype: api
   connectorType: source
   definitionId: e71aae8a-5143-11ed-bdc3-0242ac120002
-  dockerImageTag: 0.4.10
+  dockerImageTag: 0.5.0
   dockerRepository: airbyte/source-clockify
-  documentationUrl: https://docs.airbyte.com/integrations/sources/clockify
   githubIssueLabel: source-clockify
-  icon: clockify.svg
+  icon: icon.svg
   license: MIT
   name: Clockify
-  registryOverrides:
-    cloud:
-      enabled: true
-    oss:
-      enabled: true
-  releaseDate: 2023-08-27
+  releaseDate: 2025-02-08
   releaseStage: alpha
-  remoteRegistries:
-    pypi:
-      enabled: false
-      packageName: airbyte-source-clockify
   supportLevel: community
+  documentationUrl: https://docs.airbyte.com/integrations/sources/clockify
   tags:
-    - cdk:low-code
     - language:manifest-only
-  connectorTestSuitesOptions:
```

Comment:
This is a bug in our contribute flow, but @marcosmarxm be extra careful when reviewing metadata.yaml files until we fix this. 

For example, deleting CAT credential settings that are used in tests is royally not cool.